onnxruntime/test/onnx/main.cc

```diff
@@ -372,6 +390,29 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
   broken_tests.insert({"quantizelinear", "ambiguity in scalar dimensions [] vs [1]"});
 #endif
 
+#ifdef USE_OPENVINO
+  broken_tests.insert({"add_bcast", "disabled temporarily"});
+  broken_tests.insert({"averagepool_1d_default", "disabled temporarily"});
```

Comment:
do we have an understanding of why these tests are broken?
is it due to lack of support in openvino for those operators (if so, please disable via GetCapability)
or something else?

Reply:
I have removed the tests that are passing from this list and added the disabled reason for the ones that are broken